namespaces/nsenter/nsenter.c

```diff
@@ -151,6 +151,17 @@ void nsenter()
 			exit(1);
 		}
 	}
+	// blocking until the parent placed the process inside correct cgroups.
+	unsigned char s;
+	ssize_t ret;
+	do {
+		ret = read(3, &s, 1);
+	} while (ret == -1 && errno == EINTR);
```

Comment:
read() can return EINTR only if a process sets a signal handler without SA_RESTART. I think we can remove this loop from here. Thanks.